programs/zk-token-proof-tests/tests/process_transaction.rs

```diff
@@ -18,13 +21,16 @@ use {
     std::mem::size_of,
 };
 
-const VERIFY_INSTRUCTION_TYPES: [ProofInstruction; 6] = [
+const VERIFY_INSTRUCTION_TYPES: [ProofInstruction; 9] = [
     ProofInstruction::VerifyZeroBalance,
     ProofInstruction::VerifyWithdraw,
     ProofInstruction::VerifyCiphertextCiphertextEquality,
     ProofInstruction::VerifyTransfer,
     ProofInstruction::VerifyTransferWithFee,
     ProofInstruction::VerifyPubkeyValidity,
+    ProofInstruction::VerifyAggregateRangeProof64,
+    ProofInstruction::VerifyAggregateRangeProof128,
+    ProofInstruction::VerifyAggregateRangeProof256,
```

Comment:
Yes, so naming here could be a bit complicated. These instructions are basically verifying multiple range proofs in a single instruction. Doing so is more efficient (and proofs more compact) than verifying individual range proofs separately.

A suitable name could be `VerifyMultiplePedersenCommitmentsHoldSpecifiedRangeValues`, but this is a long name and there are also three variations of them depending on the sum of the bit lengths... so it would be `VerifyMultiplePedersenCommitmentsHoldSpecifiedRangeValuesWithTotalBitLength64`... I think it might be better to change the name to `VerifyBatchedRangeProofsBitLength64`. Wdyt?

Reply:
@CriesofCarrots - wdyt? Would you mind weighing in here, a 3rd pov would probably be useful

Reply:
I'm leaning toward something like `VerifyBatchedRangeProofsU64`

Reply:
Okay, I'll update to `VerifyBatchedRangeProofsU64` and then add proof description on the `zk_token_proof_instruction.rs`.